content/articles/2018-12-dplyr-0-8-0-release-candidate.Rmarkdown

```diff
@@ -15,6 +15,17 @@ photo:
   author: Pau Casals
 ---
 
+<style>
+.addendum {
+  margin-top: 20px;
+  margin-bottom: 20px;
+  padding: 10px 10px 10px 50px;
```

Comment:
I'd make the padding a constant `padding: 10px`, give the box a background (pale grey), and make the border a bit thicker.

Reply:
I used rgb(248, 248, 248) to match the bg of functions